doc/Rules.md

```diff
@@ -161,8 +161,8 @@ document:
 ```
 
 The configured list style can be a specific symbol to use (asterisk, plus, dash),
-can require that usage be consistent within the document, or can require that each
-sublist have a consistent symbol that is different from its parent list.
+can require that users be consistent within the document, or can require that each
+sublist has a consistent symbol that is different from its parent list.
```

Comment:
Same as above about "require that X be Y". This particular change should not be applied.